configs/eslint/node/typescript/index.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaVersion: 'latest',
+    project: true,
```

Comment:
We can use `parserOptions.project` to indicate which TS configuration file to use.

https://typescript-eslint.io/packages/parser#project